lib/broccoli_sass_compiler.js

```diff
@@ -288,7 +288,7 @@ BroccoliSassCompiler.prototype.compileTree = function(srcPath, files, destDir) {
   }
  return promiseChunks.reduce(function(promise, chunk) {
    return promise.then(function() {
-     return RSVP.all(chunk);
+     return RSVP.allSettled(chunk);
```

Comment:
In addition to this, you will also need to "handle" the errors in the fulfillment. What i mean by this is:

```
RSVP.allSettled([]).then(settled => {
  settled.forEach(item => {
    if (item.state === 'rejected') {
      throw item.reason; // this way if something does fail, we wait until the concurrent work is done, and then throw.
    }
  });
})
```